application_service/app_service.mak

```diff
@@ -44,13 +43,18 @@ AR=ar
 #export LD_LIBRARY_PATH=/usr/local/lib
 LDFLAGS= -L $(LOCAL_LIB) -lprotobuf -lgtest -lgflags -lpthread -L/usr/local/opt/openssl@1.1/lib/ -lcrypto -lssl
 
-dobj=	$(O)/app_service.o $(O)/certifier.pb.o $(O)/certifier.o $(O)/certifier_proofs.o $(O)/support.o \
-$(O)/simulated_enclave.o $(O)/application_enclave.o $(O)/cc_helpers.o $(O)/cc_useful.o \
-$(O)/sev_support.o $(O)/sev_report.o
+dobj = $(O)/app_service.o $(O)/certifier.pb.o $(O)/certifier.o \
+       $(O)/certifier_proofs.o $(O)/support.o $(O)/simulated_enclave.o \
+       $(O)/application_enclave.o $(O)/cc_helpers.o $(O)/cc_useful.o \
+       $(O)/sev_support.o $(O)/sev_report.o
 
-user_dobj= $(O)/test_user.o $(O)/certifier.pb.o $(O)/certifier.o $(O)/certifier_proofs.o $(O)/support.o \
-$(O)/simulated_enclave.o $(O)/application_enclave.o $(O)/cc_helpers.o $(O)/cc_useful.o
+user_dobj = $(O)/test_user.o $(O)/certifier.pb.o $(O)/certifier.o \
```

Comment:
Rearranged long-long lists of things to fit 80-char line limit which improves readability overall.